### ISSUE TITLE
Emoji not taking proper horizontal space

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#780]
Hi, a small rendering bug: it looks like there is just one space between the emoji and the text:

![image](https://user-images.githubusercontent.com/101152/42452086-611570a4-8389-11e8-8290-d1feb017243f.png)

However, there are more:

![image](https://user-images.githubusercontent.com/101152/42452021-37ebcb2e-8389-11e8-8260-b64632bb7f25.png)

I can observe this in more documents and with different emoji. This is on a Mac with latest RC.

**Comment by astoilkov** [comment]
Are you experiencing this on a non-retina display?

**Comment by borekb** [comment]
That is correct: it is on an external display. (I should have mentioned it straight away, sorry.)

**Comment by astoilkov** [comment]
This is a Chromium bug. Chromium renders the first space after an emoji on macOS on non-retina displays very small. We can workaround this issue but we were just waiting for Chromium to fix it.

**Comment by borekb** [comment]
Ah, thanks, makes sense to wait for an upstream fix.
